autogen/main.tf.tmpl

```diff
@@ -165,10 +165,9 @@ resource "google_compute_backend_service" "default" {
   project = var.project
   name    = "${var.name}-backend-${each.key}"
 
-  {% if not serverless %}{# not necessary for serverless as default port_name=http, protocol=HTTP #}
-  port_name                       = each.value.port_name
-  protocol                        = each.value.protocol
-  {% endif %}
+  port_name = lookup(each.value, "port_name", "http")
+  protocol  = lookup(each.value, "protocol", "HTTP")
+
```

Comment:
I would love to have here extra :eyes: .
should we move this default to variables? or leave it here?

one more thing: each.value.port_name vs lookup(each.value, "port_name", "HTTP")

which one is better ?